File: lambda_functions/python_lambda_functions/lambda_functions.py
import sys

square = lambda x: x**2

def main():
  if len(sys.argv) != 1:
    print("Usage: %s <noargs>" % sys.argv[0])
    sys.exit()
  for idx_i in range(1, 10):
    print ("square(%d)=<%d>" % (idx_i, square(idx_i)))
  sys.exit(), end

main()

File: lambda_functions/ocaml_lambda_functions/lambda_functions.ml
let inc = fun n -> n + 1;;

inc(2);;

let f4 (a,b,c) = a + b * c;;

f4 (2,3,4);;

let main () =
  if  Array.length Sys.argv <> 1 then
    begin
      Printf.printf "%s <noargs>\n" Sys.argv.(0);
      exit 1
    end
  else
    begin
    for idx_i = 1 to 9 do
      Printf.printf "inc(%d)=<%d>\n" idx_i (inc(idx_i))
    done;
    end

let _ = main ()
